1.7/administration/securing-your-cluster.md

```diff
@@ -29,6 +29,41 @@ the other nodes in the cluster via URL routing. For security, the DC/OS cloud
 template allows configuring a whitelist so that only specific IP address
 ranges are permitted to access the admin zone.
 
+#### Steps for Securing Admin Router
+
+By default, Admin Router will permit unencrypted HTTP traffic. This is not
+considered secure, and you must provide a valid SSL certificate and redirect
+all HTTP traffic to HTTPS in order to properly secure access to your cluster.
+
+Once you have a valid SSL certificate, install the certificate on each master.
+Edit the Admin Router `nginx.conf`, by opening the file located at
+`/opt/mesosphere/packages/adminrouter--*/nginx/conf`. Modify the nginx
```

Comment:
You can point them to `/opt/mesosphere/active/adminrouter/nginx/conf` so they don't have to worry about the package version sha